paasta_tools/paasta_maintenance.py

```diff
@@ -131,18 +135,26 @@ def get_maintenance_status():
 
     :returns: a requests.response object representing the current maintenance status
     """
-    client_fn = get_schedule_client()
+    client_fn = maintenance_api()
     return client_fn(method="GET", endpoint="/status")
 
 
+def schedule():
+    """Get the Mesos maintenance schedule. This contains hostname/ip mappings and their maintenance window.
+    :returns: None
+    """
+    schedule = get_maintenance_schedule()
+    print "%s:%s" % (schedule, schedule.text)
+
+
 def get_hosts_with_state(state):
     """Helper function to check the maintenance status and return all hosts
     listed as being in a current state
 
     :param state: State we are interested in ('down_machines' or 'draining_machines')
     :returns: A list of hostnames in the specified state or an empty list if no machines
     """
-    status = get_maintenance_status()
+    status = get_maintenance_status().json()
```

Comment:
again - maybe need to do some error handling? the api client throws an ``HTTPError`` when things go wrong doesn't it?